protocol/shannon/sanctions.go

```diff
@@ -155,7 +155,10 @@ func classifyHttpError(logger polylog.Logger, err error) (protocolobservations.S
 	}
 
 	// If we can't classify the HTTP error, it's an internal error
-	logger.Warn().Msg("Unable to classify HTTP error - defaulting to internal error")
+	logger.With(
+		"err_preview", errStr[:min(100, len(errStr))],
```

Comment:
@adshmh Can you create a global helper for `r[:min(100, len(errStr)` that we can reuse across payloads/errors?

Add a TODO to make the `100` configurable. 

Will make it easy to apply globally.